kod/include/blakston.khd

```diff
@@ -40,6 +40,14 @@
    ACTIVE = 1
    PASSIVE = 2
 
+   %%% Item text color enum
```

Comment:
The reader wants to know what an "item text color" is.

Reply:
Updated comment based on feedback.